outbox/outbox-rabbitmq/src/test/resources/config/application-postgresql.yml

```diff
@@ -7,7 +7,7 @@ spring:
         hikari:
             auto-commit: false
     jpa:
-        database-platform: org.hibernate.dialect.PostgreSQL10Dialect
```

Comment:
I remember adding the PostgreSQL10Dialect to support the implementation specific query that is used in `PostgresOutboxMessageRepository`. I'm not sure how the outbox tests are succeeding with this change

Reply:
So, when you take a look at the `PostgreSQL10Dialect`, you'll see the following:
```
public class PostgreSQL10Dialect extends PostgreSQLDialect {
    public PostgreSQL10Dialect() {
        super(DatabaseVersion.make(10));
    }
}
```

This is the constructor of `PostgreSQLDialect`"
```
public PostgreSQLDialect() {
    this(MINIMUM_VERSION); //MINIMUM_VERSION = DatabaseVersion.make(10);
}
```
That makes them equal in my book.